packages/traceability/issuer-api/src/pods/blockchain/blockchain-properties.dto.ts

```diff
@@ -1,29 +1,21 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsInt, IsNotEmpty, IsString } from 'class-validator';
 
 export class BlockchainPropertiesDTO {
     @ApiProperty({ type: Number })
-    @IsNotEmpty()
```

Comment:
Why validation is removed?

Reply:
Because it's never used as a "create" DTO, only as return values. So the validation is never really triggered, thus useless.